web/templates/index.html

```diff
@@ -218,7 +218,7 @@
     // Set up tags
     $.get("{{ api_loc }}/api/v1/tags/", function(data, status) {
       $.each(data.tags, function( index, value ) {
```

Comment:
This endpoint returns tags in the `Tags` key (notice capital `T`).

Reply:
Good catch!